pkg/operatorsource/configuring.go

```diff
@@ -114,38 +113,18 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *marketplace.O
 	packages := r.datastore.GetPackageIDsByOperatorSource(out.GetUID())
 	out.Status.Packages = packages
 
-	cscCreate := new(CatalogSourceConfigBuilder).WithTypeMeta().
-		WithNamespacedName(in.Namespace, in.Name).
-		WithLabels(in.GetLabels()).
-		WithSpec(in.Namespace, packages, in.Spec.DisplayName, in.Spec.Publisher).
-		WithOwnerLabel(in).
-		CatalogSourceConfig()
+	registryDeployer := registry.NewRegistryDeployer(r.logger, r.reader, r.client)
```

Comment:
Please maintain the same pattern here i.e. creating a `CatalogSource` instead of a `CatalogSourceConfig`.